doc/sphinx/changes.rst

```diff
@@ -14,6 +14,152 @@ Version 9.0
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
+The Rocq Prover version 9 is the first Rocq Prover release after the renaming
+from The Coq Proof Assistant. The Rocq Prover 9.0.0 is backwards compatible with
+Coq 8.20, providing compatibility shims so that developments depending on Coq
+can be easily ported, see `Porting to The Rocq Prover`_  for details. The 9.0 version
+is based on a new single binary `rocq` that dispatches commands to previously
+separate binaries, a split and renaming of the standard library to `Stdlib` and
+an internal encoding of template-polymorphism in terms of the
+more general notion of sort polymorphism.
+
+We highlight some of the most impactful changes here:
+
+  - "The Rocq Prover" is the new official name of the project. We leave to users the
+    choice of renaming their projects to reflect this change, see `Renaming Advice`_.
+    The Rocq Prover comes with a new visual identity and website, see `The Rocq Prover Website`_.
+
+  - A single `rocq` binary dispatches commands for compilation, read-eval-print-loop,
+    documentation building, dependency computation, etc... see :ref:`therocqcommands`.
+    It corresponds to the `rocq-runtime` `OCaml package <https://ocaml.org/p/rocq-runtime>`_.
+    This is a bare-bones package not depending on any Gallina code.
+    CHECK LINK
+
+  - The `Coq` standard library has been renamed and :ref:`split and <90stdlib>` into two libraries:
+
+    - A `Corelib` core library. This is an extended prelude corresponding to the `rocq-core`
+      OCaml package, which is enough to run `rocq` tactics and contains the `Ltac2` library.
+    - An `Stdlib` standard library (the `rocq-stdlib` opam package). The `Stdlib` is
+      now maintained `out of <https://github.com/rocq-prover/stdlib>`_ the main `rocq`
+      `repository <http://github/com/rocq-prover/rocq>`_.
+      FIX LINKS
```

Comment:
Maybe a word about the fact that we welcome new contributors/maintainer (call to come soon)?